src/org/zaproxy/zap/extension/websocket/WebSocketMessageDTO.java

```diff
@@ -217,17 +226,65 @@ public String getPayloadAsString(){
 			return "";
 		}
 	}
+	
+	public void setPayload(String payload) {
+	    this.payload = payload;
+	}
 
 	public boolean isForceIntercept() {
 		// Not currently supported for WebSockets
 		return false;
 	}
 
-
 	//@Override
 	public String getHeader(String arg0) {
-		// TODO Auto-generated method stub
```

Comment:
Remove the whole method? Does not seem to be used/needed (`Message` does not have it).

Reply:
yeah, makes sence